digitkraft-backend/src/main/resources/db/migration/postgres/V1.0__create_schema.sql

```diff
@@ -0,0 +1,160 @@
+CREATE TABLE addresses
```

Comment:
I added in this migrations changes for Postgres DB. When you will be modyfying existing entities remember to also update this script. You can use Spring JPA Buddy, which I highly recommend.